client/yarn.lock

```diff
@@ -4571,7 +4571,7 @@ debug@=3.1.0:
   dependencies:
```

Comment:
This file must be out of the PR. To revert it to the same version on `master`, you can run:

```bash
git checkout origin/master client/yarn.lock
```